sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,142 @@
+# February'20 release notes
+
+Release notes for commit e8f1f29
+
+## New features
+  - Added `__builtin_intel_fpga_mem` for the FPGA SYCL device. The builtin is
+    used to indicate the characteristics of the load-store unit (LSU) to be used
+    when de-referencing the pointer [1e33c01]
+  - Added support for the `intelfpga::uses_global_work_offset` attribute
+    [8bed533]
+  - Added basic xmethod implementation for `sycl::accessor::operator[]` to make
+    it callable from gdb command line [d6be8ff]
+  - Added device libraries for standard math like std::cos and `std::complex`
+    type [7abd9d5]
+
+## Improvements
+### SYCL Frontend and driver changes
+  - Added support for passing a non type template parameter to the `loop_unroll`
+    attribute [8d7a32a]
+  - Relaxed the standard layout requirement for kernel arguments. Now by default
+    they should be trivially copiable. The `sycl-std=1.2.1` driver option turns
+    standard layout requirement "on" [3adb4a5]
+  - Added diagnostic on using `__float128` type in the device code [659efdf]
+  - The `intelfpga::max_private_copies` has been renamed to
+    `intelfpga::private_copies` [97a199f]
+  - Fixed output options behavior for `-fsycl-link` on Windows [67b24d46]
+  - Prevented duplication of error diagnostics for `-fsycl` [3a0b62e]
+  - Added diagnostic when attempting to use existing FPGA static library with
+    additional device code [6431be6]
+  - Renamed `intelfpga::uses_global_work_offset` attribute to
+    `intelfpga::no_global_work_offset` [5a9058b]
+  - Added support for non-type template parameters for FPGA memory attributes
+    [ffcad03]
+
+### SYCL headers and runtime
+  - The `SYCL_DEVICE_WHITE_LIST` control was renamed to `SYCL_DEVICE_ALLOWLIST`
+    [4df18fa]
+  - Added a leaf limit to the execution graph to avoid leaf bloat in
+    applications that have an overwhelming number of command groups that can
+    be executed in parallel [7c293e2]
+  - Added `get_range()` method to the image and local accessors to align with
+    the SYCL specification [8ed5566]
+  - Added a diagnostic on attempt to create a context from devices that are
+    bound to different platforms [8f354f7]
+  - An ordered queue can now be created by passing
+   `sycl::property::queue::in_order` property to the `sycl::queue` constructor
+    [c855520]
+  - Added a diagnostic on attempt to create an accessor with an unsupported type
+    [306624e]
+  - Made host device return `nullptr` for bad USM allocation functions
+    (huge, 0, etc) [2a000d9]
+  - Added templated forms of USM allocation functions [42cf5bf]
+  - Added support for APIs that query properties of USM pointers [926e38e]
+  - Added clean up finished command nodes of the execution graph in the
```

Comment:
```suggestion
  - Added finished command nodes' clean-up of the execution graph in the
```

Reply:
```suggestion
  - Added cleanup of finished command nodes of the execution graph in the
```
I think this flows better, as in "cleanup" -> "command nodes" -> "execution graph" rather than "command nodes" <- "cleanup" -> "execution graph".

Reply:
Done.